### PR TITLE
Add Admin Inspect method

### PR DESCRIPTION
**Pull request by klauspost**
Add method for inspecting data.

### CODE REVIEW

File: inspect.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	Volume, File string
+}
+
+// Inspect makes an admin call to download a raw files from disk.
+func (adm *AdminClient) Inspect(ctx context.Context, d InspectOptions) (key [32]byte, c io.ReadCloser, err error) {
+	path := fmt.Sprintf(adminAPIPrefix + "/inspect-data")
+	resp, err := adm.executeMethod(ctx,
+		http.MethodGet, requestData{
+			relPath: path,
+			queryValues: url.Values{
```
Comment (vadmeste) [inline inspect.go:39]: Let's use golang API to work with url.Values,

```
q := make(url.Values)
q.Set("volume", d.Volume)
q.Set("file", d.File)
```

**Review by vadmeste — COMMENTED** [review]
One comment

File: inspect.go (hunk 1 above)
Comment (klauspost) [reply inspect.go:39]: Not sure I understand the difference, but no problem.

Comment (klauspost) [reply inspect.go]:
```
// Set sets the key to value. It replaces any existing
// values.
func (v Values) Set(key, value string) {
	v[key] = []string{value}
}
```
So yeah, the same AFAICT.

Comment (vadmeste) [reply inspect.go]: > So yeah, the same AFAICT.

yeah, it looks like the same :) I thought maybe Golang is doing something extra

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**